packages/backend/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "example-backend",
   "version": "0.1.1-alpha.7",
-  "main": "dist",
+  "main": "dist/index.cjs",
   "types": "src/index.ts",
   "private": true,
   "license": "Apache-2.0",
   "engines": {
     "node": ">=12"
   },
   "scripts": {
-    "build": "tsc",
-    "start": "backstage-cli watch-deps --build -- tsc-watch --onFirstSuccess \\\"nodemon -r esm\\\"",
+    "build": "backstage-cli build --outputs cjs",
+    "start": "backstage-cli watch-deps --build -- backstage-cli backend:dev",
```

Comment:
Should get rid of watch-deps now

Reply:
What will trigger the build in that case for e.g. catalog-backend if I change `/catalog-backend/src/index.ts` ?

Reply:
Oh but if we point to src, then yes! Hmm..

Reply:
Oh, that is so cool! 🤩

Reply:
Okay, not so straightforward due to how migration files are loaded during the DatabaseManager instantiation in the catalog-backend, I think maybe take it as a separate issue? @Rugvip

Reply:
what's causing trouble with loading the migrations? Is __dirname not rewritten properly?

We could do it in a later PR but I think it would be nice to do in this one as we'd just be adding a bunch of config that we then need to remove.

Reply:
migrations are .ts files with exports and getting `require`d by the knex internals.
https://github.com/knex/knex/blob/master/lib/migrate/sources/fs-migrations.js#L82

https://github.com/tgriesser/knex/issues/3003
https://github.com/tgriesser/knex/issues/2998

We can make it work by either:
1) Using `.js` migration files
2) Using `ts-node/register` in front of DatabaseManager

Reply:
Aaaand scratch all of that, here is what's gonna work: http://knexjs.org/#custom-migration-sources
Found while was writing 🙃

Reply:
Moved to #1238